### ISSUE TITLE
Dependency Check List

### ISSUE THREAD

**Issue by amittel** [nextcloud/web-installer#9]
Hi!
Just installing Nextcloud and I had to install some dependencies.
The list is not looking good. I made a screenshot, so you can have a look.
![nextcloud](https://cloud.githubusercontent.com/assets/822396/20134639/78ad2730-a66c-11e6-9bc8-4a9ea036d09d.png)

**Comment by mahe23** [comment]
I have same problem with Ubuntu 14.04 GD and CURL installed. Do you have a solution?

**Comment by amittel** [comment]
For me it was just installing ``phpX-curl`` and ``phpX-gd``

**Comment by gratuxri** [comment]
After installing php-curl and php-gd restart apache or your fastcgi daemon and reload your nextcloud installer page. It's not nextcloud issue, please close it.

**Comment by bakertaylor28** [comment]
> 
> 
> After installing php-curl and php-gd restart apache or your fastcgi daemon and reload your nextcloud installer page. It's not nextcloud issue, please close it.

No, it's very much a nextcloud issue- Optimally, Linux repositories are expected to pull and install all required dependencies via reference (as is done with apt-get) - there should be absolutely no reason to manually install a dependency, EVER. especially when using a script to do an automated install to LAMP which is supposed to overcome the need to do manual installs of anything at the terminal.  This issue kind of subverts the purpose of using the script to do an install, and should be fixed.

**Comment by tekntrash** [comment]
I second this. When you claim to have an online easy installer the minimum you can do is to tell people what commands to issue to fix a dependency issue and not just leave them to google it. 

Also, php -l-modules DOES show the packages are installed 

Garbage installer

**Comment by joshtrichards** [comment]
@bakertaylor28 The web installer is only used for installing in restrictive shared hosting environments. Usually the installer doesn't have privileges to install dependencies. It's also used across *many* different distributions. If you want an automated installation use basically any of the methods mentioned here:

  https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#installation-on-linux

@tekntrash The web installer isn't an official installer. It's a community project. Feel free to submit PRs or use one of the more modern installation methods:

  https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#installation-on-linux
